constellation_ticketbox/views.py

```diff
@@ -279,19 +285,19 @@ def api_v1_ticket_create(request, box_id):
         newTicket.box = get_object_or_404(Box, pk=box_id)
         newTicket.owner = request.user
         newTicket.anonymous = ticketForm.cleaned_data['anonymous']
-        if (newTicket.anonymous == False):
+        if (newTicket.anonymous is False):
```

Comment:
an explicit check for False here is not needed.  Check for not newTicket.anonymous instead.